string target_name
---
bool result
